gulpfile.js

```diff
@@ -20,8 +20,6 @@ var sass         = require('gulp-sass');
 var sourcemaps   = require('gulp-sourcemaps');
 var uglify       = require('gulp-uglify');
 var svgmin       = require('gulp-svgmin');
-var raster       = require('gulp-raster');
```

Comment:
These aren't needed unless the svg>png stuff is needed and thanks to modern browsers I think we're out of the woods ✋